buildtools/DevConfigGenerator/DevConfigGenerator/DevConfigFile.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Text;
+
+namespace DevConfigGenerator;
+
+public class DevConfigFile
+{
+    public CoreChange? Core { get; set; }
+
+    public List<Service> Services { get; } = new List<Service>();
+}
+
+public class CoreChange
```

Comment:
It would be useful to have `backwardIncompatibilitiesToIgnore`  but can be in a future version.

Reply:
Agreed useful but I'll leave for a future version as we see how useful this tool is.